polkadot/node/network/collator-protocol/src/collator_side/metrics.rs

```diff
@@ -151,6 +166,19 @@ impl metrics::Metrics for Metrics {
 				)?,
 				registry,
 			)?,
+			process_finalized: prometheus::register(
+				prometheus::Histogram::with_opts(
+					prometheus::HistogramOpts::new(
+						"polkadot_parachain_collator_protocol_collator_process_finalized",
```

Comment:
Why is this important ?  I'm not sure I understand what it means.

Reply:
I want to look at how much time I lose here. At least for testing burns